lib/node/rpc.js

```diff
@@ -968,7 +968,7 @@ class RPC extends RPCBase {
     if (hash) {
       block = await this.chain.getBlock(hash);
     } else if (this.chain.options.indexTX) {
-      const tx = await this.chain.getMeta(last);
+      const tx = await this.node.getMeta(last);
```

Comment:
I guess this is the difference with having indexing as a plugin on the node rather than built into the chain huh? 😄

Reply:
Yup :) Since indexing happens outside the chain.